cmd/root.go

```diff
@@ -90,6 +94,18 @@ func NewRootCmd() *cobra.Command {
 					SSHPass:                sshPass,
 					SSHKeyFile:             sshKey,
 					SSHKeyPassphrase:       sshKeyPassphrase,
+					TUIKeyBindings: command.TUIKeyBindings{
+						RunQuery: tcell.KeyCtrlSpace,
+						Navigation: command.TUINavgationBindgins{
+							Up:          tcell.KeyCtrlK,
+							Down:        tcell.KeyCtrlJ,
+							Left:        tcell.KeyCtrlH,
+							Right:       tcell.KeyCtrlL,
+							Structure:   tcell.KeyCtrlS,
+							Indexes:     tcell.KeyCtrlI,
+							Constraints: tcell.KeyCtrlT,
```

Comment:
Removing these lines enables running `go run` again. Absolutely beats my why those would be an issue 🤔

Reply:
That's on me. Haven't tried the form. It's broken, forgot to ignore the new fields on the Options struct so the application can enter in the form mode.